recipes/freetype/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.5.5" %}
+
+package:
+    name: freetype
+    version: {{ version }}
+
+source:
+    fn: freetype-{{ version }}.tar.bz2
+    url: http://download.savannah.gnu.org/releases/freetype/freetype-{{ version }}.tar.bz2
+    sha256: 387bcc4b780b12484aa2ec9f7db1a55d8286eb5639f45fbc0fbba7a4e5a1afb9
+    patches:
+        - subpixel.patch
```

Comment:
Continuing our discussion from the [outdated diff]( https://github.com/conda-forge/staged-recipes/pull/572#discussion_r62544865 ) so we can still find it. 😉

cc @ccordoba12 @ocefpaf